src/com/oltpbenchmark/DBWorkload.java

```
@@ -264,6 +264,26 @@ public static void main(String[] args) throws Exception {
                 // Nothing to do here !
             }
 
+            try {
+                wrkld.setPort(xmlConfig.getInt("port"));
+            } catch(NoSuchElementException nse) {
+                // Nothing to do here !
+            }
+
+            try {
+                wrkld.setNumDBConnections(xmlConfig.getInt("numDBConnections"));
+            } catch(NoSuchElementException nse) {
+                // Nothing to do here !
+            }
```

I don't agree with catching and ignoring exceptions. There must be a better way to get an element from an XML file. Or, if you must catch and ignore this particular exception, encapsulate that logic in a helper function taking an xml config and a key and do not wrap additional code in these exception-suppressing blocks.

---

Done